src/organizations/pipes/transform-org.pipe.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable, PipeTransform } from '@nestjs/common';
+import { UpdateOrganizationDto } from '../dto/update-organization.dto';
+import { CreateOrganizationDto } from '../dto/create-organization.dto';
+
+type T = CreateOrganizationDto | UpdateOrganizationDto;
+
+@Injectable()
+export class TransformOrgDto implements PipeTransform<T> {
+  transform(value: T) {
+    const newValue = this.convertToLowerCase(value, ['password']);
+
+    return newValue;
+  }
+
+  convertToLowerCase(obj: T, excludedKeys: string[]) {
+    const newObj = {};
+
+    for (const key in obj) {
+      if (obj.hasOwnProperty(key)) {
+        const isExcluded = excludedKeys.includes(key) || typeof obj[key] !== 'string';
```

Comment:
I believe that the only variable that we need to transform is email, and thats the only one that should be transformed. 

While I see that you ae converting to lowercase, I and not seening that you are trmming the string input, hence " abc" !== "abc "